### ISSUE TITLE
Tundra Rhino also gives charge to non beast friendly minions

### ISSUE THREAD

**Issue by Lykrast** [demilich1/metastone#250]
I was trying to mess around with with the Attribute Aura and never got managed to make it work correctly with filters. I thought it was my fault at first, but after testing, I discovered that this also applies to Tundra Rhino.

So basically, Tundra Rhino gives charge to all friendly minions, beast or not, and it seems to be a AttributeAura issue.

**Comment by webadict** [comment]
I'll check it out. I'm still under the impression we need to fix buffs
sooner or later, but it's fairly hard to change. I've got a pseudo working
version of the buff changes, but I want demilich to check it out first, so
that we can make it better.

On Sep 23, 2016 6:56 AM, "Lykrast" <notifications@github.com> wrote:

I was trying to mess around with with the Attribute Aura and never got
managed to make it work correctly with filters. I thought it was my fault
at first, but after testing, I discovered that this also applies to Tundra
Rhino.

So basically, Tundra Rhino gives charge to all friendly minions, beast or
not, and it seems to be a AttributeAura issue.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/demilich1/metastone/issues/250>, or mute the thread
